doc/lua_api.txt

```diff
@@ -453,6 +459,10 @@ the global `minetest.registered_*` tables.
 * `minetest.register_craftitem(name, item definition)`
     * added to `minetest.registered_items[name]`
 
+* `minetest.unregister_item(name)`
+    * erased from minetest.registered_items[] and from item associated table
```

Comment:
But the item is also deleted from `registered_nodes` or `registered_craftitems` or `registered_tools`, and that should be pointed out, right ? That is what I mean in "and from item associated table"